scripts/sql/61_alter_ci_template.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ci_template
+    ADD COLUMN target_platform VARCHAR(1000) DEFAULT '';
```

Comment:
If this column is not nullable then add NOT NULL constraint to it. 
Default value will insert only this value if not provided. 

not adding NOT NULL constraint to this column will result in no error for below query : 
insert into ci_template(target_platform) values (NULL).